### ISSUE TITLE
7.34 Midas stacking still works for Arc Warden clone 

### ISSUE THREAD

**Issue by Pitas3214** [ValveSoftware/Dota2-Gameplay#10656]
### Item name

Hand of Midas

### Description

The problem with multiple Midas stacks (each having a separate cooldown) is still present if the player uses an arc warden clone. Each Hand of Midas in the clone's inventory will have a separate cooldown 

### Example Match ID and Timestamp

7283758702 - 11:50 (in game match time)

**Comment by Avarise831** [comment]
#10610

**Comment by TheChetu** [comment]
Yeah I mentioned this issue in #10533 but unfortunately the OP closed it as fixed when they fixed only part of the bug, the non-functional part, but not the broken advantageous parts.
